starter-code/src/functions-and-arrays.js

```diff
@@ -1,15 +1,49 @@
 // Iteration #1: Find the maximum
+console.log(Math.max(3, 2));
+//or
+function maxTowNumbers(n1, n2){
+  if(n1 > n2){
+    return n1;
+  }else{
+    return n2;
+  }
+}
+
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+functions findLongestWord(words){
+  var longestWord = 0;
+  for(var i = 0; i < words.length; i++){
+    if ( words.length[i].length > longestWord){
```

Comment:
if (words[i].length) > longestWord) -> this code is correct, not the way you wrote, because its impossivle to get words.length[i].length